synthpops/contact_networks_industries.py

```diff
@@ -1,3 +1,9 @@
+"""
+This module uses workplace data by industry from `North American Industry Classification System`_ (NAICS) codes to modify the workplace network to include contact patterns by industry.
+
```

Comment:
"modify the workplace network to include contact patterns by industry."
 
to model workplaces as specific industries and the contact patterns for workers within each workplace.